app/helpers/paths_helper.rb

```diff
@@ -46,6 +46,6 @@ def preview_url(_edition)
 private
 
   def should_have_auth_bypass_id?(edition)
-    %w[published archived].exclude?(edition.state) && edition.temp_auth_bypass_id
+    %w[published archived].exclude?(edition.state) && edition.auth_bypass_id
```

Comment:
It'd be good to lose the auth_bypass_id conditional if we expect it to be always present.